src/Components/QuickGrid/Microsoft.AspNetCore.Components.QuickGrid/src/Columns/SortedProperty.cs

```diff
@@ -0,0 +1,20 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.AspNetCore.Components.QuickGrid;
+
+/// <summary>
+/// A ValueTuple that holds the name of a property and the direction to sort by.
```

Comment:
```suggestion
/// Holds the name of a property and the direction to sort by.
```
I think "ValueTuple" implies `System.ValueTuple`, but this is just a normal struct.